src/less/alerts.less

```diff
@@ -0,0 +1,212 @@
+/* ------------------------------------------------------------------------- *\
+    Alert Styles
+
+    1. BaseAlert ... Structural styles for all alerts
+    2. AlertStyles ... Visual styles for all alerts
+    3. AlertActions ... Actions unique to an alert
+
+    Alerts keep users informed of important and sometimes time-sensitive changes.
+    Alerts are expected to be used using the base class `.alert` and an optional
+    alert modifier class (i.e. alert--info) for enchance styling. All alerts
+    contain an "aria-describedby" role that points to the textual content of the
+    alert. When possible, an "aria-label" or "aria-labelledby" is used for to
+    describe the summary of the alert message.
+
+
+    Reference material for aria choices: https://standards.usa.gov/alerts/
+
+    Example usage:
+        <div class="alert alert--critical" aria-label="Alert description" aria-describedby="dialog1Desc">
+            <p id="dialog1Desc">This is a alert description</p>
+        </div>
+
+        <div class="alert alert--error" role="alertdialog" aria-describedby="dialog1Desc">
+            <p id="dialog1Desc">To extend your session, click the OK button</p>
+            <ul class="alert__actions">
+                <li><a href="">Continue</a></li>
+                <li><a href="">Cancel</a></li>
+            </ul>
+        </div>
+\* ------------------------------------------------------------------------- */
+@import 'measurements';
+@import 'colors';
+@import 'typography';
+
+
+
+
+
+/* ------------------------------------------------------------------------- *\
+    #BaseAlert
+
+
+    All alerts will be described with a role atribute value of "alert" or "alertdialog"
+    if the alert contains actions (see #AlertActions).
+
+    1. Clears the floats
+    2. Align border-radius, margins, and ul, p and a tag styles with bootstrap basics.
```

Comment:
I was hesitant of this as we are trying to be framework agnostic yet our current alerts are very tied in with bootstrap.  I guess these can be pulled out if necessary.